### PR TITLE
Update tr.po

### PR DESCRIPTION
**Pull request by safak45x**
Bug fixes update !!

### CODE REVIEW

File: po/tr.po [hunk 1]
```diff
@@ -47,11 +47,11 @@ msgstr "Menü"
 
 #: src/MainWindow.vala:122
 msgid "Light content"
-msgstr "Açık İçerik"
+msgstr "Açık Tema"
```
Comment (cassidyjames) [inline po/tr.po:50]: This seems to be a different translation… This should be translated as "Light content", not "Light Theme"; the idea is that it is controlling the web content, not the "theme" or style of the app itself.

File: po/tr.po [hunk 2]
```diff
@@ -47,11 +47,11 @@ msgstr "Menü"
...
 
 #: src/MainWindow.vala:123
 msgid "Dark content"
-msgstr "Karanlık İçerik"
+msgstr "Karanlık Tema"
```
Comment (cassidyjames) [inline po/tr.po:54]: Same as above

File: po/tr.po [hunk 3]
```diff
@@ -262,15 +262,15 @@ msgstr "Satın Alma veya Fon…"
...
+msgstr "Whoops"
 
 #: src/Views/ErrorView.vala:31
 msgid "Could not display the page."
 msgstr "Sayfa görüntülenemedi."
 
 #: src/Views/WelcomeView.vala:31
 msgid "The always-incognito web browser"
-msgstr "Her zaman gizli web tarayıcısı"
+msgstr "Gizli web tarayıcınız"
```
Comment (cassidyjames) [inline po/tr.po:273]: This seems to be a less direct translation, though I'm not sure if it makes more sense or not.

File: po/tr.po (hunk 1 above)
Comment (safak45x) [reply po/tr.po:50]: Sorry I thought it was related to the theme
I'm fixing

File: po/tr.po (hunk 2 above)
Comment (safak45x) [reply po/tr.po:54]: I'm fixing

File: po/tr.po (hunk 3 above)
Comment (safak45x) [reply po/tr.po:273]: More understandable such sentence
Same as narration

**Comment by safak45x** [comment]
I couldn't find where the rearrangement is done

**Review by cassidyjames — APPROVED** [review]
Thank you!

**Comment by safak45x** [comment]
> Teşekkür ederim!
Diğer uygulamalarınızı eklerseniz sevinirim

**Comment by safak45x** [comment]
> Thank you!
I would be glad if you add the translations in your other applications

**Pull request merged**